puree/src/main/java/com/cookpad/puree/outputs/PureeBufferedOutput.java

```diff
@@ -62,12 +62,14 @@ public void run() {
 
     public void flushSync() {
         if (!storage.lock()) {
+            flushTask.reset();
```

Comment:
issue https://github.com/cookpad/puree-android/issues/61
It prevent that flush will be never called. But if the issue situation loops, this approach will not be effective...
This bug might be fixed by splitting storage's lock each PureeBufferedOutput.